src/scripts/main.js

```diff
@@ -1,3 +1,263 @@
+    }
+
+    clickedSortTable = columnName;
+
+    const columnsSorted = columnsToSort.sort((a, b) => {
+      const elemA = a.children[sortedColumn].textContent;
+      const elemB = b.children[sortedColumn].textContent;
+
+      const aToNum = toNumber(elemA);
+      const bToNum = toNumber(elemB);
+
+      if (Number.isNaN(aToNum)) {
+        return isSecondClick
+          ? elemB.localeCompare(elemA)
+          : elemA.localeCompare(elemB);
+      }
+
+      return isSecondClick ? bToNum - aToNum : aToNum - bToNum;
+    });
+
+    tableBody.textContent = '';
+
+    columnsSorted.forEach((el) => {
+      tableBody.appendChild(el);
+    });
+  }
+});
+
+tableBody.addEventListener('click', (e) => {
+  const rowsChildren = [...tableBody.children];
+  const columnName = e.target.closest('tr');
+
+  rowsChildren.forEach((row) => row.removeAttribute('class'));
+  columnName.classList.add('active');
+});
+
+const formContainer = document.createElement('form');
+const nameLabel = document.createElement('label');
+const nameInput = document.createElement('input');
+const positionLabel = document.createElement('label');
+const positionInput = document.createElement('input');
+const officeLabel = document.createElement('label');
+const officeChoose = document.createElement('select');
+const ageLabel = document.createElement('label');
+const ageInput = document.createElement('input');
+const salaryLabel = document.createElement('label');
+const salaryInput = document.createElement('input');
+const formButton = document.createElement('button');
+const cityOptions = [
+  `Tokyo`,
+  `Singapore`,
+  `London`,
+  `New York`,
+  `Edinburgh`,
+  `San Francisco`,
+];
+const inputs = [nameInput, positionInput, officeChoose, ageInput, salaryInput];
+
+const modifyElement = (parentEl, child, cellName, type) => {
+  parentEl.textContent = `${cellName}:`;
+  parentEl.setAttribute('data-qa', cellName.toLowerCase());
+  child.required = true;
+  child.type = type;
+  parentEl.appendChild(child);
+};
+
+modifyElement(nameLabel, nameInput, 'Name', 'text');
+modifyElement(positionLabel, positionInput, 'Position', 'text');
+modifyElement(ageLabel, ageInput, 'Age', 'number');
+modifyElement(salaryLabel, salaryInput, 'Salary', 'number');
+
+formContainer.classList.add('new-employee-form');
+
+officeLabel.textContent = 'Office:';
+officeLabel.setAttribute('data-qa', 'office');
+officeLabel.appendChild(officeChoose);
+officeChoose.required = true;
+formButton.type = 'Submit';
+formButton.innerText = 'Save to table';
+
+cityOptions.forEach((el) => {
+  const option = document.createElement('option');
+
+  option.value = el;
+  option.innerText = el;
+  officeChoose.appendChild(option);
+});
+
+formContainer.appendChild(nameLabel);
+formContainer.appendChild(positionLabel);
+formContainer.appendChild(officeLabel);
+formContainer.appendChild(ageLabel);
+formContainer.appendChild(salaryLabel);
+formContainer.appendChild(formButton);
+
+pageBody.appendChild(formContainer);
```

Comment:
Always try to keep logic in functions and name it properly so its easy to follow what that function should do. Thanks to that our script is easier to follow.